kore/src/main/scala/org/kframework/Strategy.scala

```diff
@@ -45,6 +47,12 @@ object Strategy {
   }
 }
 
+class ContainsSCell extends ExistsK {
+  override def apply(k: KApply): java.lang.Boolean = {
+    k.klabel.name == "<s>"
```

Comment:
I think we may have a `KLabel.STRATEGY_CELL` somewhere you could use? Can you double-check?